### ISSUE TITLE
podman build -q is not completely quiet

### ISSUE THREAD

**Issue by TomSweeneyRedHat** [containers/buildah#1897]
Reported by  @Romain-Geissler-1A in https://github.com/containers/libpod/issues/3191

Hi,

Still, the output of "podman build -q" doesn't match the output of "docker build -q":

    docker outputs a single line (the final image id), while podman show all intermediate image ids
    podman prefixes it with "Using cache" when the cache was used, whereas docker only outputs the image id and nothing else.

Is it possible to match the docker output ? It's quite handy to easily fetch an image id with:
DOCKER_IMAGE_ID="$(docker build -q .)"
while with podman currently we need to filter output to have the same result.

Cheers,
Romain

Example:
```
# cat Dockerfile.http
FROM registry.access.redhat.com/ubi8/ubi:8.0

MAINTAINER Red Hat Training <training@redhat.com>
ENV DOCROOT=/var/www/html

RUN yum --disableplugin=subscription-manager --nodocs -y install httpd \
  && yum --disableplugin=subscription-manager clean all \
  && echo "Hello from the httpd-parent container!" > ${DOCROOT}/index.html

# Allows child images to inject their own content into DocumentRoot
ONBUILD COPY src/ ${DOCROOT}/

EXPOSE 80

CMD httpd -D FOREGROUND


# podman build --format=docker -t tom -q -f /root/Dockerfile.http .
--> Using cache b99dcb21a998701da6a1e7f94ea87830881f5394e09dedc2ae44c98eadf74228
--> Using cache dc2c989a735e62f27cfd9304e6d6fb742dce984098f3eca01722f71b2eb02d51
--> Using cache 8aeafcdcc3fa2d1e5d7ca346b1c23d0e408c49d908821702b560ac55c2f39231
--> Using cache e69cf420fa2517ff4a0d3f3cf6a4745c93062fa6a41a749a1bb2d201d5bffb15
--> Using cache ea872c5d8a5bb847d1bf0ae5080c15849a03cb5e33b5ff9f29198446a7a1ee57
--> Using cache f71b402b738dfca07b7545f1570962468187d6526cce27f0502f35cc9dbba432
f71b402b738dfca07b7545f1570962468187d6526cce27f0502f35cc9dbba432

```

**Comment by tylarb** [comment]
@TomSweeneyRedHat  I added the "addresses issue" as post-comment in my comment (i.e. after final ---) - so it looks like this issue didn't get closed, even though the merged commit is clearly linked to the issue. 

Probably this issue can be closed then.

**Comment by TomSweeneyRedHat** [comment]
Thanks for the heads up @tylarb, this should have closed with the merge of #1922, closing now.
